Determining if the pthread_create exist failed with the following output:
Change Dir: /home/pyp/face_app/face_platform/models/detection_alignment/cmake-build-debug/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/make" "cmTC_bacdb/fast"
/usr/bin/make -f CMakeFiles/cmTC_bacdb.dir/build.make CMakeFiles/cmTC_bacdb.dir/build
make[1]: Entering directory '/home/pyp/face_app/face_platform/models/detection_alignment/cmake-build-debug/CMakeFiles/CMakeTmp'
Building C object CMakeFiles/cmTC_bacdb.dir/CheckSymbolExists.c.o
/usr/bin/cc   -fPIC    -o CMakeFiles/cmTC_bacdb.dir/CheckSymbolExists.c.o   -c /home/pyp/face_app/face_platform/models/detection_alignment/cmake-build-debug/CMakeFiles/CMakeTmp/CheckSymbolExists.c
Linking C executable cmTC_bacdb
/home/pyp/clion-2018.1.2/bin/cmake/bin/cmake -E cmake_link_script CMakeFiles/cmTC_bacdb.dir/link.txt --verbose=1
/usr/bin/cc -fPIC     CMakeFiles/cmTC_bacdb.dir/CheckSymbolExists.c.o  -o cmTC_bacdb 
CMakeFiles/cmTC_bacdb.dir/CheckSymbolExists.c.o：在函数‘main’中：
CheckSymbolExists.c:(.text+0x1b)：对‘pthread_create’未定义的引用
collect2: error: ld returned 1 exit status
CMakeFiles/cmTC_bacdb.dir/build.make:97: recipe for target 'cmTC_bacdb' failed
make[1]: *** [cmTC_bacdb] Error 1
make[1]: Leaving directory '/home/pyp/face_app/face_platform/models/detection_alignment/cmake-build-debug/CMakeFiles/CMakeTmp'
Makefile:126: recipe for target 'cmTC_bacdb/fast' failed
make: *** [cmTC_bacdb/fast] Error 2

File /home/pyp/face_app/face_platform/models/detection_alignment/cmake-build-debug/CMakeFiles/CMakeTmp/CheckSymbolExists.c:
/* */
#include <pthread.h>

int main(int argc, char** argv)
{
  (void)argv;
#ifndef pthread_create
  return ((int*)(&pthread_create))[argc];
#else
  (void)argc;
  return 0;
#endif
}

Determining if the function pthread_create exists in the pthreads failed with the following output:
Change Dir: /home/pyp/face_app/face_platform/models/detection_alignment/cmake-build-debug/CMakeFiles/CMakeTmp

Run Build Command:"/usr/bin/make" "cmTC_06fd0/fast"
/usr/bin/make -f CMakeFiles/cmTC_06fd0.dir/build.make CMakeFiles/cmTC_06fd0.dir/build
make[1]: Entering directory '/home/pyp/face_app/face_platform/models/detection_alignment/cmake-build-debug/CMakeFiles/CMakeTmp'
Building C object CMakeFiles/cmTC_06fd0.dir/CheckFunctionExists.c.o
/usr/bin/cc   -fPIC -DCHECK_FUNCTION_EXISTS=pthread_create   -o CMakeFiles/cmTC_06fd0.dir/CheckFunctionExists.c.o   -c /home/pyp/clion-2018.1.2/bin/cmake/share/cmake-3.10/Modules/CheckFunctionExists.c
Linking C executable cmTC_06fd0
/home/pyp/clion-2018.1.2/bin/cmake/bin/cmake -E cmake_link_script CMakeFiles/cmTC_06fd0.dir/link.txt --verbose=1
/usr/bin/cc -fPIC -DCHECK_FUNCTION_EXISTS=pthread_create    CMakeFiles/cmTC_06fd0.dir/CheckFunctionExists.c.o  -o cmTC_06fd0 -lpthreads 
/usr/bin/ld: 找不到 -lpthreads
collect2: error: ld returned 1 exit status
CMakeFiles/cmTC_06fd0.dir/build.make:97: recipe for target 'cmTC_06fd0' failed
make[1]: *** [cmTC_06fd0] Error 1
make[1]: Leaving directory '/home/pyp/face_app/face_platform/models/detection_alignment/cmake-build-debug/CMakeFiles/CMakeTmp'
Makefile:126: recipe for target 'cmTC_06fd0/fast' failed
make: *** [cmTC_06fd0/fast] Error 2


